samples/interactive-tutorials/user_environment_setup.sh

```diff
@@ -43,10 +45,11 @@ export GOOGLE_APPLICATION_CREDENTIALS=~/key.json
 # install needed Google client libraries
 virtualenv -p python3 myenv
 source myenv/bin/activate
+sleep 2
 
 pip install google
 pip install google-cloud-retail
 pip install google-cloud.storage
 pip install google-cloud.bigquery
```

Comment:
This issue unrelated to your changes but please could you resolve it since you're updating the script?
```suggestion
pip install google-cloud-storage
pip install google-cloud-bigquery
```